src/chainlit/__init__.py

```diff
@@ -176,6 +176,39 @@ def db_client_factory(
     return func
 
 
+def on_file_upload(
+    accept: Union[List[str], Dict[str, List[str]]],
+    type: Optional[str] = "file",
```

Comment:
I think we shouldn't offer the option for users to change the type. I believe we should hardcode it in `config.code.on_file_upload_config`.